soc/arc/snps_emsdp/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+zephyr_compile_options(-mcpu=${GCC_M_CPU} -mno-sdata)
+
+if(CONFIG_SOC_EMSDP_EM4)
```

Comment:
Feels like this should get refactored between the other arc SoC's, but no issue.

Reply:
different core configuration has different compile options.